src/main/java/features/dicomImage/data/DicomAttributes.java

```diff
@@ -0,0 +1,16 @@
+package features.dicomImage.data;
+
+import core.data.medImage.AttributeTagType;
+import core.data.medImage.AttributeValueRepresentation;
+import core.data.medImage.MedImageAttributeMockup;
+
+/**
+ * Stores {@link MedImageAttributeMockup} for all well-known DICOM data attributes
+ */
+public final class DicomAttributes{
+    private DicomAttributes() {}
+
+    public static final MedImageAttributeMockup ONE_PIXEL_VOLUME = new MedImageAttributeMockup("onepixelvolume", "(1234, 1234)",
```

Comment:
Tag is the magic string again. We need either to document it as a constant with a clear name or add a comment here about the logic behind it